tests/xtimer_now_irq/README.md

```diff
@@ -0,0 +1,2 @@
+This test will print an error when the time is incorrect while interrups are disabled.
+This test works only reasonaly well on more than 16 Bit platforms.
```

Comment:
How bout something like: "This test must run at least over a full timer period (i.e. >72 minutes with a 32 bit timer running at 1 MHz). So it only finishes in reasonable short time on platforms with small timers like 8 bit or 16 bit."